src/Products/Annotation/Annotator/BaseAnnotator.cs

```diff
@@ -111,15 +111,18 @@ protected AnnotationReplyInfo GetAnnotationReplyInfo(CommentsEntity comment)
                 date = DateTime.Parse(comment.time);
             }
             reply.RepliedOn = date;
-            reply.UserName = comment.userName;
+            reply.User = new User() { Name = comment.userName };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372471)